logic-and-algorithm-test/3.py

```diff
@@ -1,25 +1,25 @@
 def generate_sequences(n_terms):
     # Sequence 1: Squares of natural numbers
-    seq1 = [i**2 for i in range(1, n_terms + 1)]
+    sequence1 = [i**2 for i in range(1, n_terms + 1)]
     
     # Sequence 2: n^2 - 2n + 2
-    seq2 = []
+    sequence2 = []
     for i in range (1, n_terms + 1):
         num = i**2-2*i+2
-        seq2.append(num)
+        sequence2.apd(num)
```

Comment:
Typo in method name 'apd'. It should be 'append'. This typo will cause a runtime error.